Sources/Test/SimpleTesting/Streamables/AfaTests.cs

```diff
@@ -558,6 +558,53 @@ public void AfaZeroOrOne()
             Assert.IsTrue(result.SequenceEqual(expected));
         }
 
+        [TestMethod, TestCategory("Gated")]
+        public void AfaZeroOrOneWithStartEvent()
```

Comment:
>AfaZeroOrOneWithStartEvent [](start = 20, length = 26)

Does this pass under AfaTestsWithCodegen? We haven't talked much about columnar/codegen yet, but now's a good opportunity

Reply:
1. fixed the two additional places
2. DateTime objects cannot be created with year 19229 :)
3. The tests pass in AfaTestsWithCodegen

Reply:
Let's debug 3, because I don't think it should be working.

---
In reply to: [342853624](https://github.com/microsoft/Trill/pull/116#discussion_r342853624) [](ancestors = 342853624)